common/jmeter/perf-test-is.sh

```diff
@@ -417,9 +445,10 @@ function initiailize_test() {
         mkdir results
         cp "$0" results
         mv test-metadata.json results/
-
+if [[ "$execution_method" == "Deployment_and_Test" ]] || [[ "$execution_method" == "Deployment_only" ]]; then
```

Comment:
Should be similar to below
```suggestion
if [[ "$execution_method" == "Deployment_and_Test" ]] || [[ "$execution_method" == "Test_only" ]]; then
```